app/javascript/Components/__tests__/assignment_summary.test.jsx

```diff
@@ -95,3 +95,71 @@ describe("For the AssignmentSummaryTable's display of inactive groups", () => {
     expect(screen.queryByText(/group_0001/)).not.toBeInTheDocument();
   });
 });
+
+describe("For the AssignmentSummaryTable's display of an assignment with automated testing", () => {
+  beforeEach(() => {
+    fetch.mockReset();
+    fetch.mockResolvedValueOnce(
+      Promise.resolve({
+        ok: true,
+        json: jest.fn().mockResolvedValueOnce({
+          data: [],
+          criteriaColumns: [],
+          numAssigned: 0,
+          numMarked: 0,
+          enableTest: true,
+          ltiDeployments: [],
+        }),
+      })
+    );
+
+    render(
+      <AssignmentSummaryTable
+        assignment_id={1}
+        course_id={1}
+        is_instructor={true}
+        lti_deployments={[]}
+      />
+    );
+  });
+
+  it("should render the Download Test Results button", async () => {
+    await waitFor(() => {
+      expect(screen.getByText(/download test results/i)).toBeInTheDocument();
```

Comment:
Don't hard-code the string in these tests, and instead use `I18n` here to use the localized string. This mirrors what label we assign the button in the application code